doc/fiber.md

```diff
@@ -1,16 +1,46 @@
-# Scheduler
+# Fiber
+
+Fibers provide a mechanism for cooperative concurrency.
+
+## Context Switching
```

Comment:
Can I add some text about this section after merged?

Reply:
Sure